index.html

```diff
@@ -41,49 +43,157 @@
     </div>
   </nav>
   <!-- Main Content -->
-  <div class="container">
-    <h1>My Feed</h1>
+  <h2 class="mt-5 text-center">What's on your mind, John?</h2>
+  <!-- Button trigger modal -->
+  <div class="container text-center mt-5 mb-5">
+    <div class="col-12">
+        <textarea type="button" class="btn border border-3" data-bs-toggle="modal"                   data-bs-target="#exampleModal" class="form-control" placeholder="Write here..." id="floatingTextarea2"></textarea>
+    </div>
   </div>
-  
-  <div class="container">
-    <div class="row">
-      <div class="card" style="width: 18rem;">
-        <img src="..." class="card-img-top" alt="...">
-        <div class="card-body">
-          <h5 class="card-title">Card title</h5>
-          <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
-          <a href="#" class="btn btn-primary">Go somewhere</a>
+
+  <!-- Modal -->
+  <div class="modal fade" id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h1 class="modal-title fs-5" id="exampleModalLabel">Create Post</h1>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
+        <div class="modal-body">
+          <textarea class="form-control" placeholder="What's on your mind, John?" id="floatingTextarea2" style="height: 100px"></textarea>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-primary" data-bs-toggle="modal">Post</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <h2 class="text-center mb-5">My Feed</h2>
+  <div class="container card p-3">
+    <div class="row row-cols-1 row-cols-md-3 g-4">
+      <div class="col">
+        <div class="card h-100">
+          <img src="src/images/dinner-1433494_1920.jpg" class="card-img-top" alt="...">
+          <div class="card-body">
+            <h5 class="card-title">Sunday Dinner</h5>
+            <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ratione, est.</p>
+          </div>
+          <div class="card-footer">
+            <small class="text-muted">Last updated 4 mins ago</small>
+          </div>
         </div>
       </div>
-      <div class="card" style="width: 18rem;">
```

Comment:
no custom styles.. use col to arrange content

Reply:
De som er røde er ikke lenger i koden. Det er linjer som jeg har fjernet/byttet ut. Tror jeg da :)

Reply:
Ja... selvfølgelig... er vel det.